core/templates/dev/head/pages/state_editor/StateInteractionEditorDirective.js

```diff
@@ -213,8 +215,16 @@ oppia.directive('stateInteractionEditor', [
                       UrlInterpolationService.getInteractionThumbnailImageUrl);
 
                     $scope.INTERACTION_SPECS = INTERACTION_SPECS;
-                    $scope.ALLOWED_INTERACTION_CATEGORIES = (
-                      GLOBALS.ALLOWED_INTERACTION_CATEGORIES);
+
+                    // Checks that the modal is invoked by exploration_editor or
+                    // skill_editor, topic_editor. 
+                    if ($window.location.href.indexOf('/questions') == -1) {
```

Comment:
You can use `window.location.pathname` and then match the path with the paths used in [main.py](https://github.com/oppia/oppia/blob/develop/main.py) instead of using `questions.`

Reply:
How to import the paths in main.py?

Reply:
You can use them directly here like `skill_editor`, `create` etc.

Reply:
So I need to write something like $window.location.pathname === '/skill_editor/<skill_id>/questions'. Is my understanding correct? Or do I need to write something like main.URLS_TO_SERVE?

Reply:
Fixed as directed.